Add a "How To Contribute" section to docs

See #234 and #235.

I still can't run the tests without some extra steps:

```
 ckeeney@ckeeney-Serval-WS  ~/tmp  git clone https://github.com/isaachinman/next-i18next.git
Cloning into 'next-i18next'...
remote: Enumerating objects: 17, done.
remote: Counting objects: 100% (17/17), done.
remote: Compressing objects: 100% (16/16), done.
remote: Total 2303 (delta 6), reused 1 (delta 0), pack-reused 2286
Receiving objects: 100% (2303/2303), 1.80 MiB | 3.22 MiB/s, done.
Resolving deltas: 100% (1401/1401), done.

 ckeeney@ckeeney-Serval-WS  ~/tmp  cd next-i18next
 ckeeney@ckeeney-Serval-WS  ~/tmp/next-i18next   master  yarn
yarn install v1.12.3
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
info fsevents@1.2.4: The platform "linux" is incompatible with this module.
info "fsevents@1.2.4" is an optional dependency and failed compatibility check. Excluding it from installation.
info fsevents@1.2.7: The platform "linux" is incompatible with this module.
info "fsevents@1.2.7" is an optional dependency and failed compatibility check. Excluding it from installation.
[4/5] Linking dependencies...
[5/5] Building fresh packages...
Done in 13.59s.

 ckeeney@ckeeney-Serval-WS  ~/tmp/next-i18next   master  yarn test
yarn run v1.12.3
$ yarn build:example && NODE_ENV=test jest --maxWorkers=1 --silent
$ cd examples/simple && yarn build
$ next build

● Client █████████████████████████ chunk asset optimization (92%) TerserPlugin 
 

✖ Server
  Compiled with some errors in 2.57s


✖ Client
  Compiled with some errors in 4.52s

✖ Server
  Compiled with some errors in 2.57s

ModuleNotFoundError: Module not found: Error: Can't resolve 'next-i18next/dist/commonjs' in '/home/ckeeney/tmp/next-i18next/examples/simple'
    at factory.create (/home/ckeeney/tmp/next-i18next/node_modules/webpack/lib/Compilation.js:823:10)
    at factory (/home/ckeeney/tmp/next-i18next/node_modules/webpack/lib/NormalModuleFactory.js:397:22)
    at resolver (/home/ckeeney/tmp/next-i18next/node_modules/webpack/lib/NormalModuleFactory.js:130:21)
    at asyncLib.parallel (/home/ckeeney/tmp/next-i18next/node_modules/webpack/lib/NormalModuleFactory.js:224:22)
    at /home/ckeeney/tmp/next-i18next/node_modules/neo-async/async.js:2825:7
    at /home/ckeeney/tmp/next-i18next/node_modules/neo-async/async.js:6886:13
    at normalResolver.resolve (/home/ckeeney/tmp/next-i18next/node_modules/webpack/lib/NormalModuleFactory.js:214:25)
    at doResolve (/home/ckeeney/tmp/next-i18next/node_modules/enhanced-resolve/lib/Resolver.js:184:12)
    at hook.callAsync (/home/ckeeney/tmp/next-i18next/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
    at _fn0 (eval at create (/home/ckeeney/tmp/next-i18next/node_modules/tapable/lib/HookCodeFactory.js:32:10), <anonymous>:15:1)
    at resolver.doResolve (/home/ckeeney/tmp/next-i18next/node_modules/enhanced-resolve/lib/UnsafeCachePlugin.js:37:5)
    at hook.callAsync (/home/ckeeney/tmp/next-i18next/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
    at _fn0 (eval at create (/home/ckeeney/tmp/next-i18next/node_modules/tapable/lib/HookCodeFactory.js:32:10), <anonymous>:15:1)
    at hook.callAsync (/home/ckeeney/tmp/next-i18next/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
    at _fn0 (eval at create (/home/ckeeney/tmp/next-i18next/node_modules/tapable/lib/HookCodeFactory.js:32:10), <anonymous>:12:1)
    at resolver.doResolve (/home/ckeeney/tmp/next-i18next/node_modules/enhanced-resolve/lib/DescriptionFilePlugin.js:42:38)
resolve 'next-i18next/dist/commonjs' in '/home/ckeeney/tmp/next-i18next/examples/simple'
  Parsed request is a module
  using description file: /home/ckeeney/tmp/next-i18next/examples/simple/package.json (relative path: .)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      /home/ckeeney/tmp/next-i18next/examples/simple/node_modules doesn't exist or is not a directory
      /home/ckeeney/tmp/next-i18next/examples/node_modules doesn't exist or is not a directory
      /home/ckeeney/node_modules doesn't exist or is not a directory
      /home/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory
      looking for modules in /home/ckeeney/tmp/next-i18next/node_modules
        using description file: /home/ckeeney/tmp/next-i18next/package.json (relative path: ./node_modules)
          Field 'browser' doesn't contain a valid alias configuration
      looking for modules in /home/ckeeney/tmp/node_modules
        No description file found
        Field 'browser' doesn't contain a valid alias configuration
          using description file: /home/ckeeney/tmp/next-i18next/package.json (relative path: ./node_modules/next-i18next/dist/commonjs)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              /home/ckeeney/tmp/next-i18next/node_modules/next-i18next/dist/commonjs doesn't exist
            .wasm
              Field 'browser' doesn't contain a valid alias configuration
              /home/ckeeney/tmp/next-i18next/node_modules/next-i18next/dist/commonjs.wasm doesn't exist
            .mjs
              Field 'browser' doesn't contain a valid alias configuration
              /home/ckeeney/tmp/next-i18next/node_modules/next-i18next/dist/commonjs.mjs doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
        No description file found
        no extension
          Field 'browser' doesn't contain a valid alias configuration
              /home/ckeeney/tmp/next-i18next/node_modules/next-i18next/dist/commonjs.js doesn't exist
            .jsx
              Field 'browser' doesn't contain a valid alias configuration
          /home/ckeeney/tmp/node_modules/next-i18next/dist/commonjs doesn't exist
        .wasm
          Field 'browser' doesn't contain a valid alias configuration
              /home/ckeeney/tmp/next-i18next/node_modules/next-i18next/dist/commonjs.jsx doesn't exist
            .json
              Field 'browser' doesn't contain a valid alias configuration
          /home/ckeeney/tmp/node_modules/next-i18next/dist/commonjs.wasm doesn't exist
        .mjs
          Field 'browser' doesn't contain a valid alias configuration
              /home/ckeeney/tmp/next-i18next/node_modules/next-i18next/dist/commonjs.json doesn't exist
          /home/ckeeney/tmp/node_modules/next-i18next/dist/commonjs.mjs doesn't exist
        .js
          Field 'browser' doesn't contain a valid alias configuration
            as directory
              /home/ckeeney/tmp/next-i18next/node_modules/next-i18next/dist/commonjs doesn't exist
          /home/ckeeney/tmp/node_modules/next-i18next/dist/commonjs.js doesn't exist
        .jsx
          Field 'browser' doesn't contain a valid alias configuration
          /home/ckeeney/tmp/node_modules/next-i18next/dist/commonjs.jsx doesn't exist
        .json
          Field 'browser' doesn't contain a valid alias configuration
          /home/ckeeney/tmp/node_modules/next-i18next/dist/commonjs.json doesn't exist
        as directory
          /home/ckeeney/tmp/node_modules/next-i18next/dist/commonjs doesn't exist
ModuleNotFoundError: Module not found: Error: Can't resolve 'next-i18next/dist/commonjs' in '/home/ckeeney/tmp/next-i18next/examples/simple'
    at factory.create (/home/ckeeney/tmp/next-i18next/node_modules/webpack/lib/Compilation.js:823:10)
    at factory (/home/ckeeney/tmp/next-i18next/node_modules/webpack/lib/NormalModuleFactory.js:397:22)
    at resolver (/home/ckeeney/tmp/next-i18next/node_modules/webpack/lib/NormalModuleFactory.js:130:21)
    at asyncLib.parallel (/home/ckeeney/tmp/next-i18next/node_modules/webpack/lib/NormalModuleFactory.js:224:22)
    at /home/ckeeney/tmp/next-i18next/node_modules/neo-async/async.js:2825:7
    at /home/ckeeney/tmp/next-i18next/node_modules/neo-async/async.js:6886:13
    at normalResolver.resolve (/home/ckeeney/tmp/next-i18next/node_modules/webpack/lib/NormalModuleFactory.js:214:25)
    at doResolve (/home/ckeeney/tmp/next-i18next/node_modules/enhanced-resolve/lib/Resolver.js:184:12)
    at hook.callAsync (/home/ckeeney/tmp/next-i18next/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
    at _fn0 (eval at create (/home/ckeeney/tmp/next-i18next/node_modules/tapable/lib/HookCodeFactory.js:32:10), <anonymous>:15:1)
    at resolver.doResolve (/home/ckeeney/tmp/next-i18next/node_modules/enhanced-resolve/lib/UnsafeCachePlugin.js:37:5)
    at hook.callAsync (/home/ckeeney/tmp/next-i18next/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
    at _fn0 (eval at create (/home/ckeeney/tmp/next-i18next/node_modules/tapable/lib/HookCodeFactory.js:32:10), <anonymous>:15:1)
    at hook.callAsync (/home/ckeeney/tmp/next-i18next/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
    at _fn0 (eval at create (/home/ckeeney/tmp/next-i18next/node_modules/tapable/lib/HookCodeFactory.js:32:10), <anonymous>:12:1)
    at resolver.doResolve (/home/ckeeney/tmp/next-i18next/node_modules/enhanced-resolve/lib/DescriptionFilePlugin.js:42:38)
resolve 'next-i18next/dist/commonjs' in '/home/ckeeney/tmp/next-i18next/examples/simple'
  Parsed request is a module
  using description file: /home/ckeeney/tmp/next-i18next/examples/simple/package.json (relative path: .)
    resolve as module
      /home/ckeeney/tmp/next-i18next/examples/node_modules doesn't exist or is not a directory
      /home/ckeeney/node_modules doesn't exist or is not a directory
      /home/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory
      looking for modules in /home/ckeeney/tmp/node_modules
        No description file found
      looking for modules in /home/ckeeney/tmp/next-i18next/examples/simple/node_modules
        using description file: /home/ckeeney/tmp/next-i18next/examples/simple/package.json (relative path: ./node_modules)
          using description file: /home/ckeeney/tmp/next-i18next/examples/simple/package.json (relative path: ./node_modules/next-i18next/dist/commonjs)
            no extension
              /home/ckeeney/tmp/next-i18next/examples/simple/node_modules/next-i18next/dist/commonjs doesn't exist
      looking for modules in /home/ckeeney/tmp/next-i18next/node_modules
        using description file: /home/ckeeney/tmp/next-i18next/package.json (relative path: ./node_modules)
          using description file: /home/ckeeney/tmp/next-i18next/package.json (relative path: ./node_modules/next-i18next/dist/commonjs)
            no extension
              /home/ckeeney/tmp/next-i18next/node_modules/next-i18next/dist/commonjs doesn't exist
            .wasm
              /home/ckeeney/tmp/next-i18next/examples/simple/node_modules/next-i18next/dist/commonjs.wasm doesn't exist
            .wasm
              /home/ckeeney/tmp/next-i18next/node_modules/next-i18next/dist/commonjs.wasm doesn't exist
            .js
              /home/ckeeney/tmp/next-i18next/examples/simple/node_modules/next-i18next/dist/commonjs.js doesn't exist
            .js
              /home/ckeeney/tmp/next-i18next/node_modules/next-i18next/dist/commonjs.js doesn't exist
        No description file found
            .mjs
              /home/ckeeney/tmp/next-i18next/examples/simple/node_modules/next-i18next/dist/commonjs.mjs doesn't exist
            .mjs
              /home/ckeeney/tmp/next-i18next/node_modules/next-i18next/dist/commonjs.mjs doesn't exist
        no extension
          /home/ckeeney/tmp/node_modules/next-i18next/dist/commonjs doesn't exist
            .jsx
              /home/ckeeney/tmp/next-i18next/examples/simple/node_modules/next-i18next/dist/commonjs.jsx doesn't exist
            .jsx
              /home/ckeeney/tmp/next-i18next/node_modules/next-i18next/dist/commonjs.jsx doesn't exist
        .wasm
          /home/ckeeney/tmp/node_modules/next-i18next/dist/commonjs.wasm doesn't exist
            .json
              /home/ckeeney/tmp/next-i18next/examples/simple/node_modules/next-i18next/dist/commonjs.json doesn't exist
            .json
              /home/ckeeney/tmp/next-i18next/node_modules/next-i18next/dist/commonjs.json doesn't exist
        .js
          /home/ckeeney/tmp/node_modules/next-i18next/dist/commonjs.js doesn't exist
            as directory
              /home/ckeeney/tmp/next-i18next/examples/simple/node_modules/next-i18next/dist/commonjs doesn't exist
            as directory
              /home/ckeeney/tmp/next-i18next/node_modules/next-i18next/dist/commonjs doesn't exist
        .mjs
          /home/ckeeney/tmp/node_modules/next-i18next/dist/commonjs.mjs doesn't exist
        .jsx
          /home/ckeeney/tmp/node_modules/next-i18next/dist/commonjs.jsx doesn't exist
        .json
          /home/ckeeney/tmp/node_modules/next-i18next/dist/commonjs.json doesn't exist
        as directory
          /home/ckeeney/tmp/node_modules/next-i18next/dist/commonjs doesn't exist
> Build error occurred
Error: > Build failed because of webpack errors
    at Object.build [as default] (/home/ckeeney/tmp/next-i18next/node_modules/next/dist/build/index.js:69:15)
    at process._tickCallback (internal/process/next_tick.js:68:7)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

```

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -1,0 +1,14 @@
+# Contributing
+
+## Setup local dev environment
+
+1. Clone the repo
+2. Do `yarn && yarn build && yarn build:examples/simple`
+
+## Development
+
+The `examples/simple` project is linked via `yarn link` to the main source and can be run in dev mode. Perform your desired changes to the `next-i18next` core and then run `yarn build` in the root dir to trigger live reload of the example project.
+
+## PRs
+
+All PRs must pass all checks before they will be considered for review.

